deliveries/starlay_finance-milestone_1.md

```diff
@@ -0,0 +1,20 @@
+# Milestone Delivery :mailbox:
+
+**The invoice has been send out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/smart-contract-bounty/Support-Docs/blob/master/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** [Starlay Finance](https://github.com/Chidasan/Ecosystem-Grants/blob/master/applications/starlay_finance.md).
+
+**Deliverables**
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 0a. | License | [License](https://github.com/starlay-finance/starlay-protocol-wasm/blob/main/LICENSE.md) | 　|
+| 0b. | Documentation | [Documentation](https://docs.starlay.finance) |　 |
+| 1. | Deposit function | [Deposit function](https://github.com/starlay-finance/starlay-protocol-wasm/blob/main/logics/impls/pool/mod.rs#L308)  | |
+| 2. | Borrow function | [Borrow function](https://github.com/starlay-finance/starlay-protocol-wasm/blob/main/logics/impls/pool/mod.rs#L339) | |
+| 3. | Redeem function | [Redeem function](https://github.com/starlay-finance/starlay-protocol-wasm/blob/main/logics/impls/pool/mod.rs#L322) | |
+| 4. | Repay_borrow function| [Repay_borrow function](https://github.com/starlay-finance/starlay-protocol-wasm/blob/main/logics/impls/pool/mod.rs#L35)
+
+**Additional Information**
+
+The mint function, initially stated in our application's milestone, has been removed as it was found to be redundant with the deposit function. Also please be aware that the links provided are currently private, as we have not yet conducted the audit. After the audit is completed, we will make the code public.
```

Comment:
Can you please open source the code? I don't see why this needs to be kept private until an audit is completed

Reply:
Thank you for your comment. Our code has already been made public. Initially, we chose to keep it private prior to the audit to minimize potential security risks. Also we've updated the 'Additional Information' section by removing unnecessary content. Please let us know if you have any other queries.